libraries/chain/controller.cpp

```diff
@@ -1033,7 +1033,7 @@ struct controller_impl {
       ram_delta += owner_permission.auth.get_billable_size();
       ram_delta += active_permission.auth.get_billable_size();
 
-      if (auto dm_logger = get_deep_mind_logger()) {
+      if (auto dm_logger = get_deep_mind_logger(false)) {
```

Comment:
Same comment as above.

Reply:
`create_native_account` is only called during startup, at which time we don't know transaction types.

Reply:
Right, only called during database initialization. Nevermind.